### PR TITLE
Add favourite star thing

### PR DESCRIPTION
**Pull request by realqijun**
#115

### CODE REVIEW

**Review by meerkatboy — COMMENTED** [review]
Looks good to me, good job Qi Jun!

File: src/main/resources/view/CompanyListCard.fxml [hunk 1]
```diff
@@ -26,6 +28,13 @@
           </minWidth>
         </Label>
         <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
+        <Region HBox.hgrow="ALWAYS"/>
+        <ImageView fx:id="favouriteStar" fitHeight="24" fitWidth="24"/>
+<!--        <ImageView fx:id="favoriteStar" fitHeight="24" fitWidth="24">-->
+<!--          <image>-->
+<!--            <Image url="@/images/cat2.png" />-->
+<!--          </image>-->
+<!--        </ImageView>-->
```
Comment (blanklogic) [inline src/main/resources/view/CompanyListCard.fxml:37]: Might not be good to include comments of unused code in the PR

**Review by blanklogic — APPROVED** [review]
Minor nits, loved the star you have drawn. Small change, otherwise LGTM!

**Review by meerkatboy — APPROVED** [review]
LGTM!

**Pull request merged**